app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,99 @@
+class RecipesController < ApplicationController
+  before_action :set_recipe, only: %i[show edit update destroy]
+
+  # GET /recipes or /recipes.json
+  def index
+    @user = current_user
+    @recipes = Recipe.all
+  end
+
+  # def public_recipes
+  #   @public_recipes = Recipe.includes(:user).all.where(public: true).order(created_at: :desc)
+  #   # render :public_recipes
+  # end
+
+  # GET /recipes/1 or /recipes/1.json
+  def show; end
```

Comment:
- [ ] Each recipe should have a link that leads to a details view page, well done implementing this action 👏, however, the view corresponding to this action is missing, please add that.

Reply:
I have it but I didn't commit it yet because that is for a different kanban board issue which is recipe details